aioftp/server.py

```diff
@@ -1133,7 +1133,10 @@ def _build_mlsx_facts_from_stats(self, stats):
         }
 
     async def build_mlsx_string(self, connection, path):
-        stats = await connection.path_io.stat(path)
+        try:
+            stats = await connection.path_io.stat(path)
+        except asyncio.exceptions.InvalidStateError:
```

Comment:
Almost all exceptions in path methods will be reraised as [`PathIOError`](https://github.com/aio-libs/aioftp/blob/master/aioftp/pathio.py#L88). I suggest to not use exception as flow changer. I think it is better to check that path exists, before calling stat.

Reply:
Okay, I have modified by your direction